packages/nuxt/src/core/plugins/plugin-metadata.ts

```diff
@@ -132,13 +132,17 @@ export const RemovePluginMetadataPlugin = (nuxt: Nuxt) => createUnplugin(() => {
       }
 
       let wrapped = false
+      let importName = ''
```

Comment:
would it make sense to use a Set in case user has an extraneous import - unlikely as it is? e.g. `import { defineNuxtPlugin, definePayloadPlugin as n } from '#app'`

Reply:
Thank you for reviewing. I will use a Set to exclude such edge cases.